cluster/manifests/kube2iam/daemonset.yaml

```diff
@@ -56,8 +56,8 @@ spec:
           timeoutSeconds: 3
         resources:
           limits:
-            cpu: 100m
-            memory: 100Mi
+            cpu: 25m
+            memory: 50Mi
           requests:
             cpu: 25m
-            memory: 25Mi
+            memory: 50Mi
```

Comment:
This won't work unless it's `OnDelete`. Nodes will end up having no `kube2iam` pod at all.

Reply:
fixed @aermakov-zalando

Reply:
Please fix the rest (requests are increased in other daemonsets as well).

Reply:
done, would be nice if you would tell the next time this upfront

Reply:
sorry!

Reply:
have you verified that it won't cause issues, btw?